WinUIGallery/ControlPages/XamlFundamentals/BindingPage.xaml

```diff
@@ -0,0 +1,466 @@
+                </StackPanel>
+            </core:ControlExample.Example>
+            <core:ControlExample.Xaml>
+                <x:String xml:space="preserve">
+&lt;TextBox x:Name="SourceTextBox" Width="200" PlaceholderText="Enter text here" /&gt;
+&lt;TextBox Width="200" Text="{x:Bind SourceTextBox.Text, Mode=$(BindingMode)}" PlaceholderText="Mirrors above text" /&gt;
+        </x:String>
+            </core:ControlExample.Xaml>
+            <core:ControlExample.Options>
+                <RadioButtons x:Name="BindingModeGroup"
+                              Header="Select binding mode"
+                              SelectedIndex="0"
+                              SelectionChanged="BindingModeGroup_SelectionChanged">
+                    <x:String>OneWay</x:String>
+                    <x:String>TwoWay</x:String>
+                </RadioButtons>
+            </core:ControlExample.Options>
+            <core:ControlExample.Substitutions>
+                <core:ControlExampleSubstitution Key="BindingMode"
+                                                 Value="{x:Bind BindingModeGroup.SelectedItem, Mode=OneWay}" />
+            </core:ControlExample.Substitutions>
+        </core:ControlExample>
+
+        <core:ControlExample HeaderText="Binding to a property in code-behind">
+            <core:ControlExample.Example>
+                <StackPanel Spacing="8">
+                    <TextBlock>Binding a <Bold>Slider</Bold> to update the width of a <Bold>Rectangle</Bold>.</TextBlock>
+                    <Slider x:Name="WidthSlider"
+                            Minimum="50"
+                            Maximum="300"
+                            Value="100"
+                            Width="300" />
+                    <Rectangle Width="{x:Bind WidthSlider.Value, Mode=OneWay}"
+                               Height="50"
+                               Fill="{ThemeResource SystemFillColorAttentionBrush}" />
+                </StackPanel>
+            </core:ControlExample.Example>
+            <core:ControlExample.Xaml>
+                <x:String xml:space="preserve">
+&lt;Slider x:Name="WidthSlider" Minimum="50" Maximum="300" Value="100" Width="300" /&gt;
+&lt;Rectangle Width="{x:Bind WidthSlider.Value, Mode=OneWay}" Height="50" Fill="#088" /&gt;
+        </x:String>
+            </core:ControlExample.Xaml>
+        </core:ControlExample>
+        
+        <core:ControlExample HeaderText="Using `Converter` in `x:Bind`"
+                             XamlSource="Binding\BindingSample3_xaml.txt"
+                             CSharpSource="Binding\BindingSample3_csharp.txt">
+            <core:ControlExample.Example>
+                <StackPanel Spacing="8">
+                    <DatePicker x:Name="DatePickerControl"
+                                Header="Select a date" />
+                    <TextBlock Text="{x:Bind FormatDate(DatePickerControl.SelectedDate), Mode=OneWay}" />
```

Comment:
Technically, this is a **function**, am I right? I don't see a `.. Converter="SomeConverter"`
https://learn.microsoft.com/windows/uwp/data-binding/function-bindings